.github/release-please.yml

```diff
@@ -1,2 +1,3 @@
 handleGHRelease: true
 manifest: true
+initialVersion: '0.1.0'
```

Comment:
This isn't going to work if you're using `manifest: true`. It's going to read config values from the manifest config, not here.